src/sinks/loki/config.rs

```diff
@@ -151,7 +151,14 @@ pub fn valid_label_name(label: &Template) -> bool {
         // The closest mention is in section about Parser Expression https://grafana.com/docs/loki/latest/logql/
         //
         // [a-zA-Z_][a-zA-Z0-9_]*
-        let label_trim = label.get_ref().trim();
+        //
+        // '_*' symbol at the end of the label name will be treated as a prefix for
+        // underlying object keys.
+        let mut label_trim = label.get_ref().trim();
+        if let Some(without_opening_end) = label_trim.strip_suffix("_*") {
```

Comment:
I can't think of a reason why we should *require* the field to end with `_*`. Can we just check that the field ends with `*`?

Reply:
I thought it would be better to force users to provide prefixes like the `key_` because they are easier to read. However, it seems ok to give users more flexibility on the other hand.

I arranged the code according to your suggestion. Thanks!